### ISSUE TITLE
Make ores blend in

### ISSUE THREAD

**Issue by SkyBlade1978** [MinecraftModDevelopmentMods/MinecraftMineralogy#40]
Finad a way to make different types of ore blend in with the stones they are among, either by somehow texturing them differently to blend in or consider a more scientific approach to ore placement (e.g. certain ores within certain layers).. possibly configurable.. subject for discussion!

**Comment by D3nnis3n** [comment]
I already "scientifically" place my ores through orespawn. Something like putting the ore colors on the stone texture would be nice.

**Comment by SkyBlade1978** [comment]
I think I know how to do this now, in 1.14 at least, with blockstate overrides

**Comment by YaibaToKen** [comment]
It should also be possible in 1.12, by overlaying the ore texture onto the corresponding stone texture. Just not sure if you'll need to dynamically register a block for each stone and ore combination or not :/ I remember another mod doing something like this, just not sure what mod...

**Comment by SkyBlade1978** [comment]
I would wish to avoid registering every stone type and ore combination :/

**Comment by YaibaToKen** [comment]
Yup, would get very messy, very fast. Would be h(ore)ible (bad pun, shoot me).
